resources/views/annotations/show.blade.php

```diff
@@ -13,6 +13,7 @@
     biigle.$declare('annotations.shapes', {!! $shapes !!});
     biigle.$declare('annotations.imagesIds', {!! $images->keys() !!});
     biigle.$declare('annotations.imagesFilenames', {!! $images->values() !!});
+    biigle.$declare('annotations.imagesObj', {!! $images !!});
```

Comment:
Since you only need the filenames, you can use `imagesFilenames` instead.

Reply:
I need the object, because it maps the video ids to the file names. This ensures that the screenshots get the right file name.

Reply:
I still find it redundant with `imagesIds` and `imagesFilenames` just above this line. This will inflate the generated HTML of this template with duplicate information. The image annotation tool produced a screenshot with the correct filename before, too, so there should be a way to do it with the existing information (correct me if I'm wrong).